staging/src/k8s.io/apiserver/pkg/storage/testing/watcher_tests.go

```diff
@@ -1285,6 +1321,28 @@ func RunWatchSemantics(ctx context.Context, t *testing.T, store storage.Interfac
 			podsAfterEstablishingWatch:           []*example.Pod{makePod("2")},
 			expectedEventsAfterEstablishingWatch: addEventsFromCreatedPods,
 		},
+		{
+			name:                                 "allowWatchBookmarks=true, sendInitialEvents=false, RV=unset",
+			allowWatchBookmarks:                  true,
+			sendInitialEvents:                    &falseVal,
+			initialPods:                          []*example.Pod{makePod("1")},
```

Comment:
Also - let's unify the test cases:
- let's always [in every single case] create 2 initial pods [or maybe even 3?]
- let's always create 2 pods after establishing a watch

Reply:
yes, this is a good idea, let's do that.